README.md

```diff
@@ -196,7 +196,7 @@ Go to [Netlify](https://app.netlify.com/start) and select your clone, `OK` along
 First, build the vitesse image by opening the terminal in the project's root directory.
 
 ```bash
-docker build . -t vitesse:latest
+DOCKER_BUILDKIT=1 docker build . -t vitesse:latest
```

Comment:
```suggestion
docker buildx . -t vitesse:latest
```
**Suggested**: maybe we can use `buildx` here?

Reply:
Thanks for letting me know about it